### ISSUE TITLE
Issue: Open Preview button stops working

### ISSUE THREAD

**Issue by antoninadert** [estruyf/vscode-front-matter#738]
**Describe the bug**
Preview button stops opening the Preview panel when we close the panel for the first time. 
Restarting VS Code fixes the issue

**To Reproduce**

- Open frontmatter dashboard (on Mac OS only ?)
- run the server
- Click 'Open Preview' button (it opens the preview panel)
- Close the preview panel (full-screen code is visible on screen now, and preview panel is closed)
- Try to 'Open Preview' again for the same post.
- The preview does not open any more

**Expected behavior**
The preview re-opens again

** Additional context**
Issue on 
```
Version: 1.85.2 (Universal)
Commit: 8b3775030ed1a69b13e4f4c628c612102e30a681
Date: 2024-01-18T06:40:32.531Z (5 days ago)
Electron: 25.9.7
ElectronBuildId: 26354273
Chromium: 114.0.5735.289
Node.js: 18.15.0
V8: 11.4.183.29-electron.0
OS: Darwin arm64 23.2.0
```
using Astro currently

**Comment by estruyf** [comment]
Thanks @antoninadert will take a look at what is going on and implement a fix for it.

**Comment by estruyf** [comment]
@antoninadert the issue should have been fixed in the latest beta version. Would you be able to give it a try?

**Comment by antoninadert** [comment]
It works fine.
This one seems resolved.

I noticed something else, it's that when I installed the Beta version and
uninstalled the old one, the system took a long time to show my markdown
files under the content Dashboard, even if I refreshed a lot. I don't know
why, but I tried to change the frontmatter.json to update the folders, then
I changed the "Sort by" option in the dashboard and suddenly md files
appeared.
This will probably be hard to reproduce :D

Le mar. 23 janv. 2024 à 20:24, Elio Struyf ***@***.***> a
écrit :
